### DISCUSSION TITLE
en/blog/2023/03/determination-of-moist-air-properties/

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/theme#76] [General]
# en/blog/2023/03/determination-of-moist-air-properties/

An air-water mixture is found from the heat and mass balance to be at 60°C (333 K) and 0.025 kg/kg (25 g/kg) absolute humidity. Calculate the other main paramet

http://localhost:1313/en/blog/2023/03/determination-of-moist-air-properties/

**Comment by ronibd0** [comment]
It's important to know about moist air properties inside conditioned room for laboratories or, industry product need to keep inside conditioned room for gain certain characteristics.
